source/includes/request-groovy-variables.rst

```diff
@@ -24,6 +24,9 @@
 |                        || user has logged in), created by     ||                                |
 |                        || Spring Security                     ||                                |
 +------------------------+--------------------------------------+---------------------------------+
+| pathVars               || The path variable values for the    || `Map`_                         |
```

Comment:
Maybe we should add a link to the section that explains `pathVars`? Some people might not now the concept.